server/modules/sort.py

```diff
@@ -32,11 +44,46 @@ def _do_render(table, column, is_ascending):
     2: False
 }
 
+def migrate_params(params: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    Parse deprecated sort parameters. Previously sorted 1 column,
+    with sort direction as an integrer of either 0, 1, 2 as mapped in
+    _SortAscendings.
+
+    v0:
+    params: {
+        column: 'A',
+        direction: 1
+    }
+
+    v1:
+    params: {
+        sort_columns: [
+            { colname: 'A', is_ascending: True },
+            { colname: 'B', is_ascending: False}
+        ],
+        keep_top: '2'
+    }
+
+    """
+    if 'sort_columns' not in params:
+        try:
```

Comment:
Please put this block in a separate function, `_migrate_params_v0_to_v1()`. Otherwise we'll need to move the code _later_ (when we add v2, v3, v4....) and that goes against the spirit of write-once, leave-forever.